addons/mrp/static/src/js/mrp_field_one2many_with_copy.js

```diff
@@ -0,0 +1,82 @@
+odoo.define('mrp.MrpFieldOne2ManyWithCopy', function (require) {
+
+"use strict";
+    
+var FieldOne2Many = require('web.relational_fields').FieldOne2Many;
+var ListRenderer = require('web.ListRenderer');
+var fieldRegistry = require('web.field_registry');
+var core = require('web.core');
+var _t = core._t;
+    
+var MrpFieldOne2ManyWithCopyListRenderer = ListRenderer.extend({
+
+    /**
+     * @override
+     */
+    init: function (parent, state, params) {
+        this._super.apply(this, arguments);
+        this.creates.push({
+            string: parent.nodeOptions.copy_text,
```

Comment:
I really like the fact you can set a text in the XML, and otherwise get the default text, but as this widget is bonded to the `mrp.copy.bom.operations` wizard, is it really useful ?
That could be useful if this widget could be used in multiple cases but here it looks like a bit too much.